src/samtools/samtools_index/test.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+test_dir="${meta_resources_dir}test_data"
```

Comment:
This needs a slash, right?

```suggestion
test_dir="${meta_resources_dir}/test_data"
```